README.md

```diff
@@ -167,6 +167,24 @@ asg_launch_config_key_name: ""
 asg_launch_config_instance_profile_name: ""
 ```
 
+### EFS
```

Comment:
this is not a core module and therefore documentation should be in the custom-instrastructure folder.